modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/api/lifecycle/APIVisibilityByDomainTestCase.java

```diff
@@ -206,44 +217,41 @@ public void testVisibilityForAnotherUserInSameDomainInPublisher() throws APIMana
     @Test(groups = {"wso2.am"}, description = "Test the visibility of API in Store for another user in same domain",
             dependsOnMethods = "testVisibilityForAnotherUserInSameDomainInPublisher")
     public void testVisibilityForAnotherUserInSameDomainInStore() throws APIManagerIntegrationTestException {
-        List<APIIdentifier> apiStoreAPIIdentifierList =
-                APIMTestCaseUtils.getAPIIdentifierListFromHttpResponse(apiStoreClientCarbonSuperUser2.getAPI());
-        assertTrue(APIMTestCaseUtils.isAPIAvailable(apiIdentifier, apiStoreAPIIdentifierList),
+        org.wso2.am.integration.clients.store.api.v1.dto.APIListDTO apiListDTO =
+                restAPIStoreCarbonSuperUser2.getAllPublishedAPIs();
+        assertTrue(APIMTestCaseUtils.isAPIAvailableInStore(apiIdentifier, apiListDTO),
                 "API is not visible to another user in same domain in API Store. When Visibility is private." +
                         getAPIIdentifierString(apiIdentifier));
     }
 
     @Test(groups = {"wso2.am"}, description = "Test the visibility of API in Publisher for another user in other domain",
             dependsOnMethods = "testVisibilityForAnotherUserInSameDomainInStore")
-    public void testVisibilityForAnotherUserInOtherDomainInPublisher() throws APIManagerIntegrationTestException {
-
-        List<APIIdentifier> apiPublisherAPIIdentifierList =
-                APIMTestCaseUtils.getAPIIdentifierListFromHttpResponse(
-                        apiPublisherClientAnotherUserOtherDomain.getAllAPIs());
-        assertFalse(APIMTestCaseUtils.isAPIAvailable(apiIdentifier, apiPublisherAPIIdentifierList),
-                "API is  visible to another user in other domain in API Publisher. When Visibility is private." +
-                        getAPIIdentifierString(apiIdentifier));
+    public void testVisibilityForAnotherUserInOtherDomainInPublisher() throws APIManagerIntegrationTestException,
+            ApiException {
+        APIListDTO getAllApisResponse = restAPIPublisherOtherDomainUser.getAllAPIs("carbon.super");
```

Comment:
Lets use the constant here.

Reply:
Fixed in 09e067607b550f97373afb9510ea71424bed8a67